generator/config/stage/bucket.yaml

```diff
@@ -40,3 +40,62 @@ arguments:
             A document that specifies the fields to include in the output documents in addition to the _id field. To specify the field to include, you must use accumulator expressions.
             If you do not specify an output document, the operation returns a count field containing the number of documents in each bucket.
             If you specify an output document, only the fields specified in the document are returned; i.e. the count field is not returned unless it is explicitly included in the output document.
+tests:
+    -
+        name: 'Bucket by Year and Filter by Bucket Results'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/bucket/#bucket-by-year-and-filter-by-bucket-results'
+        pipeline:
+            -
+                $bucket:
+                    groupBy: '$year_born'
+                    boundaries: [1840, 1850, 1860, 1870, 1880]
+                    default: 'Other'
+                    output:
+                        count:
+                            $sum: 1
```

Comment:
Most examples use: `$sum: 1` where `$count: {}` could be used.

Reply:
I believe this is left over from times before `$count` was introduced.

Reply:
@jmikola the docs you linked to are for the `$count` aggregation **stage**. MongoDB 5.0 introduced a `$count` aggregation **accumulator** (with the same name to make the confusion perfect) that serves as a shortcut for `{ $sum: 1 }`: [$count (aggregation accumulator)](https://www.mongodb.com/docs/manual/reference/operator/aggregation/count-accumulator/)